### PR TITLE
adds the guidelines and personal details page, button design, and con…

### PR DESCRIPTION
**Pull request by FlaIespa**
## Description

In this PR: 

- I have added the guidelines and personal details page.
- I have added the button styling to globals.css
- I have added the contributions.md file.
- I have updated the pull_request_template file. 
- I have created a simple routing from the main page to the pages I have added to facilitate the navigation of the devs to the pages I created. This will be eliminated and was also added so that every time somebody wanted to see the pages, they would not have to update the URL and simply click a button from the main page. 

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Tests
- [x] Documentation update

#### For the Guidelines and Personal Details Page
Simply run `npm run dev`  in the terminal and click the buttons that show on the main page to the guidelines and personal details page. 
Screenshot 2024-10-20 at 2 03 50 PM

#### For the pull_request_template.md 
When opening a new PR, you should automatically see the content of pull_request_template.md added to the text box. 

#### For contributions.md 
If the reproduction of the steps leads you to easily contribute to the project. 

### Guidelines Page
Screenshot 2024-10-20 at 4 34 07 PM

### Personal Details Page 
Screenshot 2024-10-20 at 4 34 58 PM
Screen at 80% so it fits within 1 screenshot

```

```
````

### CODE REVIEW

File: src/app/guidelines/page.tsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
+import React from "react";
+import { Box, Typography, Button, Container } from "@mui/material";
+
+const GuidelinesScreen: React.FC = () => {
+	return (
+		<Box
```
Comment (lewiskyron) [inline src/app/guidelines/page.tsx:6]: The containers could use some padding to create a more breathable layout and improve the overall spacing relative to the background.

File: src/app/personal-details/page.tsx [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+	const [bio, setBio] = useState("");
+	const [linkedin, setLinkedin] = useState("");
+	const [aboutMe, setAboutMe] = useState("");
+
+	const handleNext = () => {
+		// Form submission logic here
+	};
+
+	return (
+		<Box
```
Comment (lewiskyron) [inline src/app/personal-details/page.tsx:28]: Same padding comment as mentioned before.

File: src/app/personal-details/page.tsx [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+							"&:hover": {
+								backgroundColor: "#E8E8F0",
+							},
+						}}
+					>
+						<AddPhotoAlternateIcon
+							sx={{ fontSize: "32px", color: "#6C6C80" }}
+						/>
+					</IconButton>
+					<Button
```
Comment (lewiskyron) [inline src/app/personal-details/page.tsx:106]: can we maybe use tailwind or css variables for all our button configurations. Not a must for this to be merged but something to be considered by the front-end team just to standardize the design.

File: src/app/personal-details/page.tsx [hunk 4]
```diff
@@ -0,0 +1,260 @@
...
+						color: "#000000",
+						marginBottom: "32px",
+						fontSize: "24px",
+						lineHeight: "30.24px",
+						fontFamily: "'Outfit', sans-serif",
+						fontWeight: 200,
+					}}
+				>
+					This helps other founders understand who you are when reviewing your
+					ideas.
```
Comment (lewiskyron) [inline src/app/personal-details/page.tsx:77]: I think the text weight is to much. Maybe something less heavy. Something to be discussed with the designers.

**Review by lewiskyron — COMMENTED** [review]
The front-end look really good! Thanks so much for the quick iteration given the new color palette that was later introduced.

File: src/app/personal-details/page.tsx (hunk 3 above)
Comment (lewiskyron) [reply src/app/personal-details/page.tsx:106]: Ignore the comment.

File: src/app/personal-details/page.tsx (hunk 4 above)
Comment (FlaIespa) [reply src/app/personal-details/page.tsx:77]: got it! let's leave like this for now and discuss with the designers.

**Pull request merged**